File: atv10/exerc10.go
package main
import "fmt"

func SomarDois(a int, b int) int {
    return a + b
}

func SomarTres(a int, b int, c int) int {
    return a + b + c
}

func main() {
    fmt.Println(SomarDois(2, 3))
    fmt.Println(SomarTres(1, 2, 3))
}

File: atv10/exerc10.py
class Calculadora:
    def somar_dois(self, a, b):
        return a + b

    def somar_tres(self, a, b, c):
        return a + b + c

calc = Calculadora()
print(calc.somar_dois(2, 3))
print(calc.somar_tres(1, 2, 3))
